doc/rados/troubleshooting/troubleshooting-mon.rst

```diff
@@ -1,115 +1,128 @@
+  selected monitor. In certain cases, however, it might be helpful to connect
+  to a specific monitor (or to several specific monitors in sequence) by adding
+  the ``-m`` flag to the command: for example, ``ceph -s -m mymon1``.
 
-  If ``ceph -s`` hangs without obtaining a reply from the cluster
-  or showing ``fault`` messages, then it is likely that your monitors
-  are either down completely or just a fraction are up -- a fraction
-  insufficient to form a majority quorum.  This check will connect to an
-  arbitrary mon; in rare cases it may be illuminating to bind to specific
-  mons in sequence by adding e.g. ``-m mymon1`` to the command.
 
-**What if ceph -s doesn't come back?**
+**None of this worked. What now?**
 
-  If you haven't gone through all the steps so far, please go back and do.
+  If the above solutions have not resolved your problems, you might find it
+  helpful to examine each individual monitor in turn. Whether or not a quorum
+  has been formed, it is possible to contact each monitor individually and
+  request its status by using the ``ceph tell mon.ID mon_status`` command (here
+  ``ID`` is the monitor's identifier).
 
-  You can contact each monitor individually asking them for their status,
-  regardless of a quorum being formed. This can be achieved using
-  ``ceph tell mon.ID mon_status``, ID being the monitor's identifier. You should
-  perform this for each monitor in the cluster. In section `Understanding
-  mon_status`_ we will explain how to interpret the output of this command.
+  Run the ``ceph tell mon.ID mon_status`` command for each monitor in the
+  cluster. For more on this command's output, see `Understanding mon_status
+  <rados_troubleshoting_troubleshooting_mon_understanding_mon_status>`.
 
-  You may instead SSH into each mon node and query the daemon's admin socket.
+  There is also an alternative method: SSH into each monitor node and query the
+  daemon's admin socket. See `Using the Monitor's Admin Socket
+  <rados_troubleshoting_troubleshooting_mon_using_admin_socket>`.
 
+.. _rados_troubleshoting_troubleshooting_mon_using_admin_socket:
 
 Using the monitor's admin socket
-=================================
+================================
 
-The admin socket allows you to interact with a given daemon directly using a
-Unix socket file. This file can be found in your monitor's ``run`` directory.
-By default, the admin socket will be kept in ``/var/run/ceph/ceph-mon.ID.asok``
-but this may be elsewhere if you have overridden the default directory. If you
-don't find it there, check your ``ceph.conf`` for an alternative path or
-run::
+A monitor's admin socket allows you to interact directly with a specific daemon
+by using a Unix socket file. This file is found in the monitor's ``run``
+directory. The admin socket's default directory is
+``/var/run/ceph/ceph-mon.ID.asok``, but this can be overridden and the admin
+socket might be elsewhere. If you cannot find it, either check your
```

Comment:
elsewhere, especially if your cluster's daemons are deployed in containers.

Reply:
Accepted. Changed.